Makefile

```diff
@@ -2,7 +2,7 @@
 #
 # You can set these variables from the command line.
 
-SPHINXOPTS    = -W --keep-going -q -T
+SPHINXOPTS    = -W --keep-going -T
```

Comment:
Is this change intended?

Reply:
No, I'll undo it later. It's impossible for me to debug in quiet mode so I make that change. I accidentally commited it.

Reply:
Ok undid that change